asset-transfer-basic/chaincode-java/README.md

```diff
@@ -0,0 +1,160 @@
+
+## Basic asset transfer 
+
+This project demonstrates the use of the Java SDK, running a basic asset transfer contract using the "chaincode as a service" 
+pattern.
+
+
+## Setup 
+
+In this sample we will employ the [Kubernetes Test Network](../../test-network-k8s) to illustrate a scenario of 
+building, running, and debugging chaincode on a development workstation.  
+
+This project is also compatible with the legacy chaincode builder pipeline and the compose based test-network.
+For additional details, see the [End-to-end with the test-network](../../test-network/CHAINCODE_AS_A_SERVICE_TUTORIAL.md#end-to-end-with-the-the-test-network)
+documentation.
+
+## [Quickstart](../../test-network-k8s#quickstart)
+
+```
+export PATH=${PWD}/../../test-network-k8s:$PATH 
+
+network kind 
+```
+```
+network up 
+network channel create
+```
+```
+network chaincode deploy    asset-transfer-basic basic_1.0 ${PWD}
+```
+```
+network chaincode metadata  asset-transfer-basic
+network chaincode invoke    asset-transfer-basic '{"Args":["InitLedger"]}'
+network chaincode query     asset-transfer-basic '{"Args":["ReadAsset","asset1"]}' | jq 
+```
+
+## Detailed Guide 
+
+```shell
+network down
+network up
+network channel create
+```
+
+```shell
+# Build the chaincode docker image 
+docker build -t fabric-samples/asset-transfer-basic/chaincode-java .
+
+# Load the docker image directly to the KIND control plane.  
+# (Alternately, build/tag/push the image to a remote container registry, e.g. localhost:5000) 
+kind load docker-image fabric-samples/asset-transfer-basic/chaincode-java
```

Comment:
My computer refuses to type "entrypoint" without auto-spell-correcting to "entryxoint."

The corollary is true:  If the computer can bother you, it will :^)